frontend/src/pages/article/{StrapiArticle.slug}.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { graphql } from "gatsby";
 import Img from "gatsby-image";
 import Moment from "react-moment";
-import Layout from "../components/layout";
+import Layout from "../../components/layout";
```

Comment:
ℹ️ the important part here is to move the template to the page folder. Check the name of the file 😊